test/WebSites/CustomUIConfig/Startup.cs

```diff
@@ -73,14 +73,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 // Other
                 c.DocumentTitle = "CustomUIConfig";
                 c.StylesPath = "/ext/custom-stylesheet.css";
-                c.ScriptBundlePath = "/ext/custom-javascript.js";
-                c.ScriptPresetsPath = "/ext/custom-javascript.js";
```

Comment:
Why did you remove these?

Reply:
Then you need to add a different test, because you're changing our test coverage and it's also going to break this test:

https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/d7a92a121d6c3c22c3d880b75c05dda52832a819/test/Swashbuckle.AspNetCore.IntegrationTests/SwaggerUIIntegrationTests.cs#L132-L151